lib/streamlit/elements/dialog_decorator.py

```diff
@@ -79,17 +79,17 @@ def dialog_content() -> None:
 
 
 @overload
-def dialog_decorator(title: str, *, width: DialogWidth = "small") -> Callable[[F], F]:
-    ...
+def dialog_decorator(
+    title: str, *, width: DialogWidth = "small"
+) -> Callable[[F], F]: ...
```

Comment:
we could think about replacing `...` with `pass` in context of this PR to get also rid of the code scanning error